### PR TITLE
add doc for wiping etcd state

### PR DESCRIPTION
**Pull request by jamiehannaford**
Fixes https://github.com/kubernetes/kubeadm/issues/362

/cc @luxas

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5939)

### CODE REVIEW

**Comment by aanm** [comment]
@jamiehannaford small nit, is the same CLI command for etcd api v2 and api v3?

**Comment by jamiehannaford** [comment]
@aanm I don't think so. v3 is generally favoured AFAIK, but there's a link on their README (which we link to) which points people to the v2 API if they're still using it

**Comment by aanm** [comment]
@jamiehannaford since they are not the same command, can you add the command in the Note added?

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -461,6 +461,13 @@ kubeadm reset
 If you wish to start over simply run `kubeadm init` or `kubeadm join` with the
 appropriate arguments.
 
+**Note**: `kubeadm reset` will not delete any etcd data. This means that if you run 
+`kubeadm init` using the same etcd endpoints, you will see state from previous 
+clusters. To wipe etcd data after reset, it is recommended you use a client like 
+`etcdctl`, such as `etcdctl del "" --prefix`. See 
+[their documentation](https://github.com/coreos/etcd/tree/master/etcdctl) for more 
+information.
+
```
Comment (steveperry-53) [inline docs/setup/independent/create-cluster-kubeadm.md]: Suggested change to text:

To wipe etc data after a reset, use a client like
[etcdctl](https://github.com/coreos/etcd/tree/master/etcdctl).
Here's an example of a command that wipes etcd data:

```shell
etcdctl del ''" --prefix
```

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 2]
```diff
@@ -461,6 +461,13 @@ kubeadm reset
 If you wish to start over simply run `kubeadm init` or `kubeadm join` with the
 appropriate arguments.
 
+**Note**: `kubeadm reset` will not delete any etcd data. This means that if you run
```
Comment (luxas) [inline docs/setup/independent/create-cluster-kubeadm.md]: **if external etcd is used**. In case of local etcd, everything's wiped out.

Comment (jamiehannaford) [reply docs/setup/independent/create-cluster-kubeadm.md]: good catch

**Comment by steveperry-53** [comment]
Ping @jamiehannaford

**Comment by jamiehannaford** [comment]
@luxas @aanm Does this lgtm to you?

**Comment by steveperry-53** [comment]
@jamiehannaford Are you ready for me to merge this? Or do you want to add a WARNING like @luxas suggests?

**Comment by jamiehannaford** [comment]
@steveperry-53 No, it's fine as is. Merge away!

**Comment by wyde** [comment]
Thank you guys for the hard work. However, what I've download following by this instruction https://kubernetes.io/docs/setup/independent/high-availability/#kubeadm-init-master0 it's still for API version: 2, and it's in a cluster.

[root@k8s]# etcdctl --version
etcdctl version: 3.1.10
API version: 2

And I bumped into the same problem which I need to wipe out all the data on this external etcd clusters whenever I need to kubeadm reset.

Can I suggest the doc to have command for API version: 2 and for the cluster like the deployment guide in the above-metioned link propose?

**Pull request merged**